org-code-javabuilder/protocol/src/main/java/org/code/protocol/LoggerUtils.java

```diff
@@ -33,6 +33,18 @@
     }
   }
 
+  /**
+   * Log a warning-level message for the given exception
+   *
+   * @param e
+   */
+  public static void sendWarningForException(Exception e) {
```

Comment:
Nice! There are a couple spots in `CodeBuilderRunnable` where I think we could use this, but we can consolidate that later.

Reply:
Realized this PR is sort of moot after your change, since we now log for gone exceptions. I think I'll abandon this one, but we can add this warning helper later if it would help!